deployment/CE02SHSP_Deploy.csv

```diff
@@ -314,12 +314,21 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
-36,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00281,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
-36,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12876,2024-05-02T20:35:00,,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00007,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00346,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00672,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01207,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00870,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00283,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00504,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00281,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+36,CE02SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-12876,2024-05-02T20:35:00,2024-08-14T19:08:00,44.63413,-124.30132,CGMCE-02SHSP-00036,EK20240502,CGVEH-CSPPCE-00007,1,,,,,80,80,,,
+37,CE02SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00002,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00336,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00352,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01519,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00425,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
+37,CE02SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00138,2024-08-30T17:57:00,2024-09-18T16:25:00,44.63611,-124.30050,CGMCE-02SHSP-00037,EK20240830,CGVEH-CSPPCE-00002,1,,,,,83,83,,,
```

Comment:
The last OPTAA-J calibration I see for SN 0138 in AM is 2021.  Are there more recent calibrations to enter?

Reply:
Indeed, last cal